### DISCUSSION TITLE
SnapshotTesting failed to build (Extra argument 'sourceLocation' in call)

### DISCUSSION THREAD

**Discussion by kostasniks** [pointfreeco/swift-snapshot-testing#918] [General]
**Describe the bug**
I've seen in the past someone else also [reported](https://github.com/pointfreeco/swift-snapshot-testing/issues/868) this and I thought it was solved but it keeps giving me this.
The problem is that SnapshotTesting Library is not building under Xcode 16.0 beta 2 (16A5171r).

**To Reproduce**
Xcode 16.0 beta 2 (16A5171r)

**Expected behavior**
Build without errors.

**Screenshots**
![CleanShot 2024-10-11 at 00 33 33](https://github.com/user-attachments/assets/86cab7f4-a26c-4cce-bccd-90044b33a027)

**Environment**
  - swift-snapshot-testing version 1.17.5 (7b0bbbae90c41f848f90ac7b4df6c4f50068256d)
  - Xcode  Xcode 16.0 beta 2 (16A5171r)
  - OS: 14.5 MacOS iOS 17.5

**Comment by mbrandonw** [comment]
Hi @kostasniks, you are using a very old beta of Xcode. If you use the official release this error will not appear.

Since this isn't an issue with the library I am going to convert it to a discussion. Please feel free to continue the conversation over there!
